### ISSUE TITLE
java.io.IOException: Not a valid TensorFlow Graph serialization: NodeDef expected inputs '' do not match 1 inputs specified; Op<name=Const; signature= -> output:dtype; attr=value:tensor; attr=dtype:type>; NodeDef: dropout_4/cond/dropout/random_uniform/max = Const[dtype=DT_FLOAT, value=Tensor<type: float shape: [] values: 1>](dropout_4/cond/Switch:1)

### ISSUE THREAD

**Issue by KMKnation** [anuradhacse/MachineLearningRepo#1]
I have done following things and gets above error.
even model is unused in export_model so why its there?

```
model = load_model('models/improved model/model.h5')
print(K.get_session().graph.get_operations())

#get your modes input output node through debugger
# input = 'conv2d_1_input'
# output = 'dense_2/Softmax'

export_model(tf.train.Saver(), model, ["conv2d_1_input"], "dense_2/Softmax")
```
